rita_db_migration/src/lib.rs

```diff
@@ -94,3 +100,25 @@ pub fn get_database_connection(
         }
     }
 }
+
+pub async fn db_migration_user_admin(
+    web3_url: String,
+    state_admin_key: PrivateKey,
+    user_to_add: Address,
+    db_addr: Address,
+) {
+    let contact = Web3::new(&web3_url, WEB3_TIMEOUT);
+
+    // We need to be a user admin to add users
+    let res = add_user_admin(
+        &contact,
+        db_addr,
```

Comment:
we should check if we are already a user admin before adding ourselves. As this will error if we're already a user admin.

Reply:
Fixed